cmd/rookd/main.go

```diff
@@ -46,6 +46,9 @@ var rootCmd = &cobra.Command{
 }
 var cfg = newConfig()
 
+var logLevelRaw string
+var logger = capnslog.NewPackageLogger("github.com/rook/rook", "main")
```

Comment:
the convention is to use repo and pkg name, so i went with main since that's the actual package name.  it could be either one though.